hw/ip/edn/data/edn.hjson

```diff
@@ -433,6 +435,8 @@
       swaccess: "rw",
       hwaccess: "hro",
       hwqe: "true",
+      tags: [// Internal HW can modify status register
```

Comment:
this comment does not seem right, since this is an error code test register.

Reply:
updated comment to reflect not wanted a fatal alert.